### PR TITLE
Documentation for Data Models

### PR DESCRIPTION
**Pull request by blueandgold**

### CODE REVIEW

File: _docs/_latest/concepts/models.md [hunk 1]
```diff
@@ -5,4 +5,80 @@ order: 100
...
+listed in the `models` table.  
+
+### binding_members
+
+This table contains information about members, and the `binding_id` of what
+it's bound to in the `bindings` table.
+
+### bindings
+
+This table contains information about the binding id for a resource,
```
Comment (joecheuk) [inline _docs/_latest/concepts/models.md]: `binding id` is bit unclear here.

File: _docs/_latest/concepts/models.md [hunk 2]
```diff
@@ -5,4 +5,80 @@ order: 100
...
+create your own set of data model, and then delete them after finish using
+Explain.
+
+Also important, the Scanner and Explainer components depend on the data models
+as the data source, i.e. it is a pre-requiste for the data models to be
+created successfully, before the Scanner or the Explainer can be used.
+
+## Tables
+
+Each set of data models are stored in its own set of tables, which are named
```
[the remark this answers is not in the document]
Comment (blueandgold) [reply _docs/_latest/concepts/models.md]: Done.

File: _docs/_latest/concepts/models.md [hunk 3]
```diff
@@ -5,4 +5,80 @@ order: 100
...
+with the `<model_handle>_<table_name>` and are tied to each other by specific
+relationships.  At any given time, multiple set of tables can exist,
+either created by the cron job, or by other users. The table sets are
+listed in the `models` table.  
+
+### binding_members
+
+This table contains information about members, and the `binding_id` of what
```
[the remark this answers is not in the document]
Comment (blueandgold) [reply _docs/_latest/concepts/models.md]: Rewording this.

File: _docs/_latest/concepts/models.md (hunk 1 above)
Comment (blueandgold) [reply _docs/_latest/concepts/models.md]: Rewording this.

**Pull request merged**